cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/elements/NodeConversion.java

```diff
@@ -68,34 +68,30 @@ private VoltageLevel newBoundarySubstationVoltageLevel() {
         String substationName = "boundary";
         String vlName = "boundary";
         return context.network()
-                .newSubstation()
-                .setId(context.namingStrategy().getId("Substation", substationId))
-                .setName(substationName)
-                // TODO(mathbagu): Country should be optional. This will be done in another PR.
-                // A non-null country code must be set
-                // This is an arbitrary country code, Bangladesh code BD also matches with
-                // BounDary
-                .setCountry(boundaryCountryCode())
-                .add()
-                .newVoltageLevel()
-                .setId(context.namingStrategy().getId("VoltageLevel", vlId))
-                .setName(vlName)
-                .setNominalV(nominalVoltage)
-                .setTopologyKind(context.nodeBreaker() ? TopologyKind.NODE_BREAKER : TopologyKind.BUS_BREAKER)
-                .add();
```

Comment:
@zamarrenolm you should configure your IDE to set to 8 spaces the continuous indentation.